events/0316_git.html

```diff
@@ -0,0 +1,215 @@
+    <link rel="stylesheet" href="css/theme/gdiaa.css" id="theme">
+    <link rel="stylesheet" href="lib/css/rainbow.css">
+    <link rel="stylesheet" href="css/print/pdf.css" media="print">
+
+    <script src="lib/js/head.min.js"></script>
+
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+</head>
+<body>
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Title -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to Version Control<br/><small><span class="green">Git, GitHub, &amp; the Terminal</span></small></h3>
+                    <p><small>James York &nbsp; | &nbsp; @kroysemaj &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #git-intro</small></p>
+                </div>
+            </section>
+
+            <!-- GDI Welcome -->
+            <section>
+                <h3>Welcome!</h3>
+                <p class="box"><strong>Girl Develop It</strong> is here to empower and assist women interested in getting into and advancing in the tech industry through affordable and accessible programs.</p>
+                <!-- <p class="box"><strong>Girl Develop It</strong> is here to provide affordable programs for adult women interested in learning web and software development in a judgment-free environment.</p> -->
+
+                <!-- Uncomment for Workshops -->
+                <div class="copy--small">
+                    <h4 class="green">Some "rules"</h4>
+                    <ul>
+                        <li>We are here for you!</li>
+                        <li>Every question is important.</li>
+                        <li>Help each other.</li>
+                        <li><strong>Have fun!</strong></li>
+                    </ul>
+                </div>
+
+                <p class="box copy--small"><small><strong>Girl Develop It is dedicated to providing a harrasment free learning experience for <em>everyone</em>.</strong><br />For more information, see our <a href="http://www.girldevelopit.com/code-of-conduct" target="_blank">Code of Conduct</a>.</small></p>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Workshop Slides</h3>
+                <div class="box"><a target="_blank" href="http://bit.ly/#">bit.ly/#</a></div>
+            </section>
+
+            <!-- Link to Slides -->
+            <section>
+                <h3>Slack!</h3>
+                <div class="box"><a target="_blank" href="https://gdiaa.slack.com/messages/0316-git-workshop/team/">#0316-git-workshop</a></div>
+                <p><small>Not on Slack? <a target="_blank" href="http://bit.ly/gdiaa-slack">bit.ly/gdiaa-slack</a></small></p>
+            </section>
```

Comment:
let's move the workshop link and slack info slides to just before the closing slide, this just allows us to leave them up for a little longer than we'd be able to with them in the beginning here